include/screencast_common.h

```diff
@@ -0,0 +1,107 @@
+#ifndef SCREENCAST_COMMON_H
+#define SCREENCAST_COMMON_H
+
+#include <pthread.h>
+#include <pipewire/pipewire.h>
+#include <spa/param/video/format-utils.h>
+#include <libdrm/drm_fourcc.h>
+#include <wayland-client-protocol.h>
+
+// Disableable logger
+//#define logger(...) printf(__VA_ARGS__)
+#define logger(...)
```

Comment:
Maybe we should establish a proper logger that could can enable/disable with a CLI flag.

Reply:
Fixed.